template_notebook.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "domestic-remove",
+   "metadata": {},
+   "source": [
+    "(template_notebook)=\n",
+    "# This is a template notebook\n",
+    "\n",
+    ":::{post} January, 2023\n",
+    ":tags: binomial regression, generalized linear model, \n",
+    ":category: beginner\n",
+    ":author: Jane Doe\n",
+    ":::"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "72588976-efc3-4adc-bec2-bc5b6ac4b7e1",
+   "metadata": {},
+   "source": [
+    "This is some introductory text. Consult the [style guide](https://docs.pymc.io/en/latest/contributing/jupyter_style.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "elect-softball",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import arviz as az\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pymc as pm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "level-balance",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_format = 'retina'\n",
```

Comment:
I am not sure this is needed but I don't know what it's supposed to do either

Reply:
It's not needed, but it sets the plot resolution to be high, appropriate for those with high res displays (eg on macs). I'll add a comment clarifying this, but would be ok with removing it. But preference is to keep... standard low res stuff looks bad on Apple monitors.